test/integration/targets/nxos_nxapi/tests/cli/configure.yaml

```diff
@@ -22,13 +22,13 @@
         - show nxapi | json
     register: result
 
-  - include: targets/nxos_nxapi/tasks/platform/n7k/assert_changes_https.yaml
+  - include: tasks/platform/n7k/assert_changes_https.yaml
```

Comment:
Thinking about this more and I believe its best not make any changes to tests under `targets/nxos_nxapi` or `targets/nxos_nxapi/transports`.  We should still have some coverage for nxapi local until the code is removed from  `module_utils/nxos.py`.  These two test suites are specific to nxapi validation so these make sense to keep around.